packages/web/app/components/FormattedTableCell.jsx

```diff
@@ -38,10 +38,12 @@ const HeadCellWrapper = styled.div`
 `;
 
 function round(float, { rounding } = {}) {
-  if (isNaN(float) || !isNumber(rounding)) {
+  const floatNumber = parseFloat(float);
+  if (isNaN(floatNumber) || !isNumber(rounding)) {
```

Comment:
This feels like a much more defensive approach. Calling straight `isNaN` returns `false` on a surprisingly large number of things (eg. `isNaN(null) = false`, `isNaN(true) = false`, etc.)
Doing a `parseFloat` first will mean it only works for numbers and strings of numbers

Reply:
I was so surprised that null and true return false but yeah